{"name": "test_multiple_login_page_success[invalid_user1-wrong_password]", "status": "passed", "parameters": [{"name": "username", "value": "'invalid_user1'"}, {"name": "password", "value": "'wrong_password'"}], "start": 1753502341448, "stop": 1753502342821, "uuid": "72f959cb-6d9e-4c3c-a4c0-080b565e3b83", "historyId": "67e9264d245abd41856c5586200953c3", "testCaseId": "90d4f72dd78db4ae114ce1f1cc9587a0", "fullName": "tests.test_multiple_login#test_multiple_login_page_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_multiple_login"}, {"name": "host", "value": "ahmads-MacBook-Pro.local"}, {"name": "thread", "value": "53821-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_multiple_login"}], "titlePath": ["tests", "test_multiple_login.py"]}